service/storage_service.cc

```diff
@@ -667,6 +667,7 @@ future<> storage_service::topology_state_load() {
     co_await _sl_controller.invoke_on_all([this] (qos::service_level_controller& sl_controller) {
         sl_controller.upgrade_to_v2(_qp, _group0->client());
     });
+    co_await update_service_levels_cache();
```

Comment:
I do not understand the commit message. It says "it needs to be
initialized on group0 start". This, here, is not group0 start.

Reply:
You're right, fixed the commit message.